shuup/utils/multilanguage_model_form.py

```diff
@@ -54,6 +55,7 @@ def __init__(self, **kwargs):
         self.trans_field_map = defaultdict(dict)
         self.trans_name_map = defaultdict(dict)
         self.translated_field_names = []
+        self.required_if_translated_field_names = []
```

Comment:
can't this just be named `required_translated_field_names`? Also minor but if order doesn't matter, this should be a set.